mm2src/coins/rpc_command/tendermint/staking.rs

```diff
@@ -47,7 +47,8 @@ impl From<TendermintCoinRpcError> for StakingInfoError {
         match e {
             TendermintCoinRpcError::InvalidResponse(e)
             | TendermintCoinRpcError::PerformError(e)
-            | TendermintCoinRpcError::RpcClientError(e) => StakingInfoError::Transport(e),
+            | TendermintCoinRpcError::RpcClientError(e)
+            | TendermintCoinRpcError::NotFound(e) => StakingInfoError::Transport(e),
```

Comment:
Here `TendermintCoinRpcError::NotFound` is defended as Transport error

But for `BalanceError` and for `ValidatePaymentError` I see the matching to InternalError (`tendermint_coin.rs` file)
<img width="1147" alt="image" src="https://github.com/user-attachments/assets/58adb5e4-4258-450f-92a9-e9fd45a3927b" />

Reply:
Fixed.